doc/material-getting-started.md

```diff
@@ -10,14 +10,64 @@ Uno Material lets you apply [Material Design 3](https://m3.material.io/) styling
 > If you are updating Uno.Material to v2 from an older 1.x version of the package, additional steps will be required. Refer to the [Uno Material Migration Guide](material-migration.md).
 
 ## Getting Started
-1. Open an existing Uno project, or create a new Uno project using the `Uno Platform App` template.
-2. In the Solution Explorer panel, right-click on your solution name and select `Manage NuGet Packages for Solution ...`. Choose [`Uno.Material`](https://www.nuget.org/packages/Uno.Material/) package for Uno UWP solution or choose the [`Uno.Material.WinUI`](https://www.nuget.org/packages/Uno.Material.WinUI) package for Uno WinUI solution. Then, select the following projects to include it:
+
+### `MaterialTheme`
+
+There is now a new, simplified method for initializing the Material Toolkit resources. It is recommended to use the new `MaterialTheme` going forward since `MaterialResources`, `MaterialColors`, and `MaterialFonts` have been deprecated.
```

Comment:
Saying "new" in the documentation is not really useful here, since tomorrow it will not be new anymore ;) 

This looks like an upgrade note, should it be present in the section below?